flight/PiOS/inc/pios_can.h

```diff
@@ -35,6 +35,7 @@
 
 //! The set of CAN messages
 enum pios_can_messages {
+	PIOS_CAN_ERROR = -1,
```

Comment:
@kubark42 I'd probably rather just update the code that validates the msg_id. It's actually not entirely safe anyway since it doesn't check that msg_id is too great:

```
bool pios_can_valid_msg(enum pios_can_messages msg_id)
{
    return (msg_id >= 0 && msg_id < PIOS_CAN_LAST);
}
```

Then in (for example PIOS_CAN_TxData)
```
int32_t PIOS_CAN_TxData(uintptr_t id, enum pios_can_messages msg_id, uint8_t *data)
{
	// Fetch the size of this message type or error if unknown
	int32_t bytes = get_message_size(msg_id);
	if (! pios_can_valid_msg(msg_id))
		return -1;

```

I can prepare a commit if you'd like. Since this is the only place we have to define a negative enum for this reason, I'd rather just fix it up stream.